Document new rate limiter settings (#489)
directus/docs: content/configuration/security-limits.md

| `RATE_LIMITER_EMAIL_ENABLED`                | Whether or not to enable rate limiting for emails.                      | `false`       |
| `RATE_LIMITER_EMAIL_POINTS`                 | The amount of allowed emails per duration.                              | `60`          |
| `RATE_LIMITER_EMAIL_DURATION`               | The time window in seconds in which the points are counted.             | `60`          |
| `RATE_LIMITER_EMAIL_QUEUE_SIZE`             | The amount of items that will be queued before erroring.                | `1000000`     |
| `RATE_LIMITER_EMAIL_ERROR_MESSAGE`          | A custom error message which is appended to the rate limit error.       | `''`          |

### Email Flows Operation Rate Limiting

You can use the built-in email rate-limiter. works the same as the api rate limiter, if you hit the limit then the flow operation starts erroring and dropping the mails.

| Variable                                    | Description                                                             | Default Value |
| ------------------------------------------- | ----------------------------------------------------------------------- | ------------- |

> | `RATE_LIMITER_EMAIL_FLOWS_ENABLED`          | Whether or not to enable rate limiting per IP on the API.               | `false`       |

Reviewer: ```suggestion
| `RATE_LIMITER_EMAIL_FLOWS_ENABLED`          | Whether or not to enable rate limiting for the `Send Email` operation.               | `false`       |
```
